packages/forma-36-react-components/src/components/Autocomplete/Autocomplete.stories.tsx

```diff
@@ -39,12 +40,20 @@ export const Basic = (args: AutocompleteProps<{}>) => {
     [setFilteredItems],
   );
 
+  const handleChange = useCallback(
+    (item: Item) => {
+      setSelectedItem(item.label);
+    },
+    [setSelectedItem],
+  );
```

Comment:
Thats a good question! I will try if it works without, I asked myself the same, but as `handleQueryChange` was using "useCallback" i tried to follow this pattern.

Reply:
yep! I saw it in the old code, but since you are touching the component I used the opportunity to ask you a favor hahahahaha 

Thanks for checking this 🙏

Reply:
I looked up the Hooks Documentation again, to see if its not some sort of optimisation. https://reactjs.org/docs/hooks-reference.html#usecallback 

But as the reference variable is the set function, i do not see how it would prevent an unnecessary update. So i think its best so remove the hook and just have the arrow function.

Reply:
Exactly what you said!

It's not optimising anything. In fact, it's making it less performant because it calls `useCallback`
[this article by Kent C. Dodds](https://kentcdodds.com/blog/usememo-and-usecallback) explains it what I think it's happening (or not happening actually hahahaha)

Reply:
I guess when this component was built we were still learning all these peculiarities of react hooks